microsoft/botframework-solutions
# [TypeScript] Update botbuilder dependencies

## Description

- Update `botbuilder-*` dependencies to `4.4.0`
- Update `cards`
- Update `npm-shrinkwrap` file

Close #1297 

## Testing Steps
**Prerequisite: update the libraries of `myget` with these changes**

Using the generator for `Virtual Assistants` and `Skills`
1. Generate a `customAssistant`
1. Generate a `customSkill`
1. Run `npm install`
1. Run `npm run build`
1. Run `npm run start`
1. Verify that the new card is shown
1. Verify the behavior of the bot with these changes
